dask/array/tests/test_cupy_reductions.py

```diff
@@ -56,15 +56,15 @@ def test_nanarg_reductions(dfunc, func):
     assert_eq(dfunc(a), func(x))
     assert_eq(dfunc(a, 0), func(x, 0))
     with pytest.raises(ValueError):
-        with pytest.warns(None):  # All NaN axis
+        with pytest.warns(RuntimeWarning):  # All NaN axis
```

Comment:
This changes the meaning of the test to testing that a `RuntimeWarning` is raised (which we don't actually care about. `pytest.warns(None)` was used before to silence all warnings, not to test anything specific about the warnings. See my top-level comment for a recommended fix.